optaplanner-core/src/main/java/org/optaplanner/core/api/score/buildin/bendable/BendableScore.java

```diff
@@ -149,18 +149,6 @@ public int getLevelsSize() {
         return hardScores.length + softScores.length;
     }
 
-    /**
-     * @param level {@code 0 <= level <} {@link #getLevelsSize()}
-     * @return higher is better
-     */
-    public int getHardOrSoftScore(int level) {
```

Comment:
Why is this method removed? It can be quite useful to users (although it might be not used by our examples or ourselves yet).